spec/datetimeformat.html

```diff
@@ -278,7 +283,7 @@
             1. Let _v_ be the value of _tm_'s field whose name is the Internal Slot column of the matching row.
             1. If _p_ is *"year"* and _v_ ≤ 0, let _v_ be 1 - _v_.
             1. If _p_ is *"month"*, increase _v_ by 1.
-            1. If _p_ is *"hour"* and _dateTimeFormat_.[[Hour12]] is *true*, then
+            1. If _p_ is *"hour"* and _dateTimeFormat_.[[HourCycle]] is *true*, then
```

Comment:
since I don't want to add other hour cycle formats yet, I'll just switch it to 'h12'.